src/manifest.json

```diff
@@ -35,13 +35,13 @@
     "css": ["styles.css"],
     "run_at": "document_end",
     "js": ["all-frames.js"],
-    "matches": ["https://csportal24.u4a.se/p470014-web/*"]
+    "matches": ["https://ubw.u4a.se/P470014-web/*"]
```

Comment:
Maybe we should just have `https://ubw.u4a.se/*`? That pattern might be less likely to need updating in the future, and I'm guessing that everything under `ubw` is time tracking